.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1        
+jobs:
+  test-arm:
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.medium
+    environment:
+      # Change to pin rust versino
```

Comment:
Typo

Reply:
This was simply copied from master. Feel free to open a PR that fixes it on master.